### ISSUE TITLE
Time zone issue with Onlyoffice in Nextcloud

### ISSUE THREAD

**Issue by akang1798** [ONLYOFFICE/onlyoffice-nextcloud#1004]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Docker

### Version information

8.1.0.169

### Expected Behavior

Expected Date and Time: 
2024-07-09 15:55:44

### Actual Behavior

Actual Date and Time Displayed on Watermarks:
2024-07-09 07:55:44
![图片](https://github.com/ONLYOFFICE/DocumentServer/assets/82021637/91ff7f63-7388-456b-b3f9-ff091a7d0e6b)

### Additional information

- Version information
Operating System: Rocky Linux 9.3
Docker Version: V26.1.4
Nextcloud Version: v28.0.7
Onlyoffice Version: 8.1.0.169

- Nextcloud and Onlyoffice are installed and working properly under Docker on Rocky Linux 9.3.

- The watermark feature is enabled in the ONLYOFFICE settings in Nextcloud.
![图片](https://github.com/ONLYOFFICE/DocumentServer/assets/82021637/9912d885-4a3a-45fc-9a31-ad5c338b1ff0)

- The watermark contains the date variable, which is displaying the incorrect time.

- The user's time zone is UTC+8 (East Eight Time Zone).

- Changing the time zone in /etc/onlyoffice/documentserver/default.json and restarting the container does not resolve the issue.
```shell
 "damengExtraOptions": {
        "columnNameUpperCase": false,
        "columnNameCase": "lower",
        "connectTimeout": 60000,
        "loginEncrypt": false,
-	"localTimezone": 0,
+	"localTimezone": 8,
        "poolTimeout": 60,
        "socketTimeout": 60000,
        "queueTimeout": 60000
},
```

**Comment by igwyd** [comment]
Thank you for the report, created ticket #69105

**Comment by akang1798** [comment]
> Hello @akang1798, this is the NextCloud's server time, not documentserver. But this is implemented in our connector, do you want the user's time to be displayed?

Yes, I want the watermark and other Onlyoffice-related times to be in my local time.
I checked the Nextcloud server time, and it matches my actual time zone.
![图片](https://github.com/ONLYOFFICE/onlyoffice-nextcloud/assets/82021637/b9f02369-1bfc-4ffc-a289-7f77d8aaa1d3)

**Comment by igwyd** [comment]
Hello @akang1798, its fixed at https://github.com/ONLYOFFICE/onlyoffice-nextcloud/commit/0bfb7fbe498efc27047b2986948340b11c9bc985, and available in onlyoffice-nextcloud connector v9.4.0 - now the watermark time settings will depend on the settings Availability -> Time zone.

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.
